### ISSUE TITLE
Remove InferenceData object warning from v4

### ISSUE THREAD

**Issue by twiecki** [pymc-devs/pymc#4740]
Currently `v4` still displays:
```
FutureWarning: In v4.0, pm.sample will return an `arviz.InferenceData` object instead of a `MultiTrace` by default. You can pass return_inferencedata=True or return_inferencedata=False to be safe and silence this warning.
```

when calling `pm.sample()`. Need to remove and set `return_inferencedata=True`.

**Comment by michaelosthege** [comment]
The issue exists already and is part of the v4 milestone: #4372

**Comment by michaelosthege** [comment]
Closed by 0923d255a45773b7a92bbe36cb11ad5d946a3122
